fix: update IndexBuilder color palette

bodom0015 opened the pull request:
## Problem
The current purple color scale for the IndexBuilder is confusing - users have trouble telling whether high or low values are"worse"

Fixes #180 

## Approach
* fix: adjust color scale for the Index Builder so that it goes from blue (low vulnerability) -> yellow (neutral) -> red (high vulnerability)

![Screenshot 2024-07-24 at 1 05 05 PM](https://github.com/user-attachments/assets/2eed1212-bc8e-414a-8965-0f670979109e)

## How to Test
1. Navigate to /builder
2. Select one or more variables from which to build a Custom Index, and click Next
3. Assign weights to each selected indicator, and click Next
4. View the map visualization for your Custom Index
    * You should see that the color scale now goes from blue (less vulnerable) to yellow (mildly vulnerable) to red (more vulnerable)

Makosak commented:
This is looking great!! It's a lot easier to interpret now.

Can we add some helper text somewhere, indicating _higher_ scores show increasing vulnerability? Either as an information icon in the legend, and/or as default text within the sidebar window?

Makosak commented:
Should we call it "Custom Vulnerability Index" ??

bodom0015 commented:
That seems reasonable! I have added a bit of descriptive text to the panel on the left
I also changed "Custom Index" -> "Custom Vulnerability Index" throughout :+1:

![Screenshot 2024-07-29 at 10 18 39 AM](https://github.com/user-attachments/assets/57936155-6b9d-46ad-b6c9-117dc3d3fb4b)

Last minor things (on the `/map` view):
* fixed the styling error in the Data Description - restored `<p>` as a child of `<div>` (but noting that this comes with an innocuous console error)
* changed "Try with multiple variables"  to say "Create a Custom Vulnerability Index using multiple variables"
![Screenshot_2024-07-29_at_10_23_45_AM](https://github.com/user-attachments/assets/54736338-d1ee-44c3-912d-cf0e7199e7d8)
